src/components/pages/MoviesPage/Movies.tsx

```diff
@@ -82,18 +84,27 @@ export const Movies = () => {
     setIsVisible(false);
   };
 
-  const openModal = async (e: React.MouseEvent<HTMLImageElement>) => {
-    setIsVisible(true);
+  const getVideoData = async () => {
     const { data } = await axios.get(
-      `https://api.themoviedb.org/3/movie/${e.currentTarget.id}/videos?api_key=15c32b97f897fcdcf60aac8f6e0746f4&language=en-US`
+      `https://api.themoviedb.org/3/movie/${currentId}/videos?api_key=15c32b97f897fcdcf60aac8f6e0746f4&language=en-US`
```

Comment:
move https://api.themoviedb.org/3/movie into constants and make the call in redux

Reply:
moved to constants and made call with redux